content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,730 @@
+the form, then click :guilabel:`Save` when done.
+
+Edit column form
+~~~~~~~~~~~~~~~~
+
+The :guilabel:`Edit Column` form is where the stage's settings are configured. The only required
+fields are the :guilabel:`Stage Name`, and the :guilabel:`Tooltips` section.
+
+The fields to be populated or modified are:
+
+- :guilabel:`Stage Name`: Type in a name for the stage.
+- :guilabel:`Email Template`: Select an email template to be used from the drop-down menu. If a
+  template is selected, when the applicant card enters the stage, an email will automatically be
+  sent to the applicant using the selected template.
+- :guilabel:`Folded in Kanban`: Check the box to have the stage appear folded (hidden) at all times
+  in the default view.
+- :guilabel:`Hired Stage`: Check the box if this stage indicates that the applicant is hired. When
+  an applicant's card enters this stage, the card displays a :guilabel:`Hired` banner in the corner.
+  If this box is checked, this stage is used to determine the hire date of an applicant.
+- :guilabel:`Job Specific`: If the stage only applies to a specific job position(s), select the job
+  position(s) from the drop-down menu. Multiple job positions can be selected.
+- :guilabel:`Show in Referrals`: Check the box if this stage should be seen in the *Referrals*
+  application, and allow the referrer to accrue points when a referral of theirs reaches this stage.
+- :guilabel:`Tooltips` section: There are three pre-configured labels (colored circles) for each
+  applicant's card, indicating its status. The *names* for the label can be modified, but the label
+  itself (the color) cannot. The default names and labels are: :guilabel:`In Progress` (gray),
+  :guilabel:`Blocked` (red), and :guilabel:`Ready for Next Stage` (green).
+- :guilabel:`Requirements`: Enter any internal notes for this stage explaining any requirements of
+  the stage.
+
+Delete stage
+------------
+
+If a stage is no longer needed, the stage can be deleted. To delete a stage, hover over the name of
+the stage, and a :guilabel:`⚙️ (gear)` icon appears. Click on the :guilabel:`⚙️ (gear)` icon and a
+drop-down menu is visible. Click :guilabel:`Delete` in the drop-down menu, and a pop-up warning
+appears, asking :guilabel:`Are you sure you want to remove this column?` Click :guilabel:`OK` to
+delete.
+
+.. image:: recruitment/delete-stage.png
+   :align: center
+   :alt: Delete a stage by accessing the drop-down menu from the gear icon.
+
+.. Important::
+   If there are applicants currently in the stage being deleted, an error pops up when attempting to
+   delete the stage, explaining that there are records in the stage which need to be deleted,
+   archived, or moved to a different stage before deleting.
+
+Email templates
+===============
+
+To communicate with the applicant, Odoo has several pre-configured email templates that can be used.
+The pre-configured email templates and when to use them are as follows:
```

Comment:
I added this second info, since the other info is in a detailed doc about adding a new job. I was having a hard time after the fleet doc deciding which info should be in which doc to prevent doubling up of info.